elytron/src/main/java/org/wildfly/extension/elytron/ModifiableKeyStoreDecorator.java

```diff
@@ -100,10 +112,22 @@ protected void executeRuntimeStep(OperationContext context, ModelNode operation)
             try {
                 ModelNode result = context.getResult();
                 Enumeration<String> aliases = keyStore.aliases();
+                ModelNode aliasNode = null;
+                boolean verbose = true;
+                String alias = null;
+                boolean recursive = false;
                 while (aliases.hasMoreElements()) {
-                    result.add(aliases.nextElement());
+                    alias = aliases.nextElement();
+                    recursive = RECURSIVE.resolveModelAttribute(context, operation).asBoolean();
```

Comment:
This should be moved outside the `while` loop since the value of `RECURSIVE` doesn't change during the loop.